Esclarecendo alguns pontos e corrigindo outros poucos, sobre licenciamento

aurium opened the pull request:
Update licencas.md

* O que são licenças.
* FS x OSS.
* Copyleft.
* Domínio Público.
* Atualização dos exemplos.

aurium committed:
Update licencas.md

Esclareci alguns pontos e corrigi outros poucos, sobre licenciamento.

* O que são licenças.
* FS x OSS.
* Copyleft.
* Domínio Público.
* Atualização dos exemplos.

lanjoni commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
 
 ### MIT
-- O código-fonte não precisa ser público.
+- O código-fonte não precisa ser públicado ou entregue com os binários, apesar de ser um contracenso que inutiliza boa parte da licença.
```
Redundância de termos apresentados (repetindo várias vezes o mesmo ponto). O intuito do guia é ser para iniciantes, para uma primeira certificação.

jpellegrini replied:
Remover o acento agudo em "públicado"

lanjoni commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
+Uma licenças FOSS exigem que o código-fonte de um software esteja disponível e apesar da confusão comum, não impedem a venda ou cobrança para modificações, manutenção, entre outros serviços que podem ser prestados sob o uso do software em si, ou atrelados à algum determinado produto. Uma licença que proibe isso é a CC-by-nc (Creative Commons - Non Comercial). Essa licença não é considerada FOSS ( ...
 
-Podemos dizer que todo Software Livre é Software de Código Aberto, afinal, uma de suas requisições para ser livre é ter seu código-fonte aberto, mas, nem todo Software de Código Aberto é Livre.
+A diferença entre FS e OSS está no discurso, vide as falas de [https://www.gnu.org/philosophy/open-source-misses-the-point.html](Stallman) e [https://lists.debian.org/debian-devel/1999/02/msg01641.html](Perens). Suas definições estao relacionadas: A definição de Open Source (em 10 pontos) veio da DFSG (Debian Free Software Gidelines), que é um "detalhamento" das 4 liberdades do SL. Ou seja, todos ...
```
O intuito do projeto é ser um conteúdo introdutório, simples, de âmbito geral para pessoas que nunca tiveram sequer algum contato com estes conceitos ou com Linux na vida. A construção de conhecimento acontece em partes, na qual, tendo em vista a preparação para a certificação LPI LE-1, esta informação é desnecessária.

Os conceitos de Software Livre e Open Source já foram bem definidos e já se enquadram nas características preparatórias de uma certificação.

jpellegrini commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
 ## O que são licenças?
 
-Licenças são as limitações que um software pode possuir em seu uso com relação ao usuário. Basicamente podemos dizer que as licenças são responsáveis por definir até onde o usuário pode ir utilizando um determinado software. Sua importância é na determinação dos direitos de propriedade de um software, isto é, definir claramente o que o usuário pode fazer com aquele software.
+Licenças são como contratos de uso, onde o detentos do copyright (em geral o autor) define o que um usuário pode ou não fazer com um produto protegido pelas leis de propriede intelectual. Basicamente podemos dizer que as licenças são responsáveis por definir até onde o usuário pode ir utilizando um determinado software. Sua importância é na determinação dos direitos de propriedade de um software, ...
```
Typo: "detentos" -> "detentor"

lanjoni replied:
Resolvido em https://github.com/lanjoni/lpi4noobs/pull/2/commits/66ab045e005211009f778d4cf773fcbf00a7a10c

jpellegrini commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
+Software sob licenças FOSS **não** são o mesmo que softwares em "domínio público". As licenças se apoiam na existência do Copyright, mesmo que subvertendo a sua prática padrão e são uma clara normatização do uso, enquanto obras em domínio público são protegidas apenas para manutenção da autoria. (Você não pode dizer que criou a Mona Lisa.)
 
-Uma licença de código aberto garante que o código-fonte de um software seja aberto, mas, permite custos para modificações, manutenção, entre outros serviços que podem ser prestados sob o uso do software em si, atrelados à algum determinado produto. 
+Uma licenças FOSS exigem que o código-fonte de um software esteja disponível e apesar da confusão comum, não impedem a venda ou cobrança para modificações, manutenção, entre outros serviços que podem ser prestados sob o uso do software em si, ou atrelados à algum determinado produto. Uma licença que proibe isso é a CC-by-nc (Creative Commons - Non Comercial). Essa licença não é considerada FOSS ( ...
```
Sobrou o "Uma" no singular, no início do parágrafo. Sugestão: "Uma licenças" -> "Licenças"

lanjoni replied:
Resolvido em https://github.com/lanjoni/lpi4noobs/pull/2/commits/a56219c768ff251730eff06f8a253cc5880a346b

jpellegrini commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
 
 ### Apache
-- O código-fonte não precisa ser público.
+- O código-fonte não precisa ser públicado ou entregue com os binários, apesar de ser um contracenso que inutiliza boa parte da licença.
```
Remover o acento agudo em "públicado"

jpellegrini commented on content/intro/licencas.md:
```diff
@@ -2,46 +2,55 @@
 
 ### BSD (Berkeley Software Distribution)
-- O código-fonte não precisa ser público.
+- O código-fonte não precisa ser públicado ou entregue com os binários, apesar de ser um contracenso que inutiliza boa parte da licença.
```
Remover o acento agudo em "públicado"

lanjoni committed:
fix: typo for licencas.md

# Resolvendo problemas de gramática

- Typo: "detentos" -> "detentor"
- Remover o acento agudo em "públicado"

Obrigado @jpellegrini e @aurium

lanjoni committed:
fix: typo for `Licenças`

# Adicionando correção para início do parágrafo

- Sobrou o "Uma" no singular, no início do parágrafo. Sugestão: "Uma licenças" -> "Licenças"

Obrigado @jpellegrini
